tests/antithesis/test-template/robustness/main.go

```diff
@@ -98,7 +97,10 @@ func testRobustness(ctx context.Context, hosts, persistedRequestdirs []string, r
 		panic(err)
 	}
 	lg.Info("Start traffic generation", zap.Duration("duration", duration))
-	reports := runTraffic(ctx, lg, hosts, baseTime, duration)
+	r := report.TestReport{Logger: lg}
+
+	lg.Info("Start traffic generation")
```

Comment:
Duplicate with 3 lines above. Looks like a bad rebase.

Reply:
Yep, it was from the rebase :( sorry about that.